### DISCUSSION TITLE
Jeffries-Matusita distance

### DISCUSSION THREAD

**Discussion by sermomon** [semiautomaticgit/SemiAutomaticClassificationPlugin#210] [General]
SCP is able to compute spectral separability statistics, including Jeffries-Matusita (JM) distance. However, SCP provides a single JMdist value for each class combination, rather than one JMdist for each class combination and each spectral band.

How is this global value calculated? Is it the average of all bands?

**Comment by semiautomaticgit** [comment]
Hello @sermomon , each spectral signature is calculated as a vector. Please read https://semiautomaticclassificationmanual.readthedocs.io/en/latest/remote_sensing.html#jeffries-matusita-distance
